dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/security/config/DhisWebApiWebSecurityConfig.java

```diff
@@ -272,25 +272,25 @@ private void configureAccessRestrictions(
           .permitAll()
           .antMatchers(apiContextPath + "/**/loginConfig")
           .permitAll()
-          .antMatchers(apiContextPath + "/auth/login")
+          .antMatchers(apiContextPath + "/**/auth/login")
           .permitAll()
-          .antMatchers(apiContextPath + "/authentication/login")
+          .antMatchers(apiContextPath + "/**/authentication/login")
           .permitAll()
-          .antMatchers(apiContextPath + "/account/recovery")
+          .antMatchers(apiContextPath + "/**/account/recovery")
           .permitAll()
-          .antMatchers(apiContextPath + "/account/restore")
+          .antMatchers(apiContextPath + "/**/account/restore")
           .permitAll()
-          .antMatchers(apiContextPath + "/account")
+          .antMatchers(apiContextPath + "/**/account")
           .permitAll()
-          .antMatchers(apiContextPath + "/staticContent/**")
+          .antMatchers(apiContextPath + "/**/staticContent/**")
           .permitAll()
-          .antMatchers(apiContextPath + "/externalFileResources/**")
+          .antMatchers(apiContextPath + "/**/externalFileResources/**")
           .permitAll()
-          .antMatchers(apiContextPath + "/icons/*/icon.svg")
+          .antMatchers(apiContextPath + "/**/icons/*/icon.svg")
```

Comment:
Hm.. is this now (or soon) conflicting with the "custom" icons under `/api/icons/{key}` ?